owncloudApp/src/main/java/com/owncloud/android/shares/presentation/ShareActivity.kt

```diff
@@ -567,6 +589,14 @@ class ShareActivity : FileActivity(), ShareFragmentListener {
         fileOperationsHelper.copyOrSendPublicLink(share)
     }
 
+    private fun updatePublicShareFile(isSharedViaLink: Boolean) {
+        val file = storageManager.getFileByPath(file.remotePath)
```

Comment:
you can refactor this block as well as suggested before ;-)

Reply:
Done